cygnus-ngsi/spec/SPECS/cygnus-ngsi.spec

```
@@ -99,6 +99,15 @@ rm -rf $RPM_BUILD_ROOT
 %{_project_install_dir}
 
 %changelog
+* Fri Sep 4 2020 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.4.0
```

```suggestion
* Fri Sep 04 2020 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.4.0
```

(Not sure but in older entries we did that way...)